README.md

```diff
@@ -13,3 +17,16 @@ Options:
       --raw       Display the raw Markdown page without formatting
   -h, --help:     Display this help
 ```
+
+## Installation 
+
+```
+make build 
+sudo make install
```

Comment:
If we woud write 
```
install: build
 ...
```
in the Makefile we could skip the `make build` step, right?

Reply:
Then we will need sudo for g++, some people will prefer to not run sudo with that. Better to make it 2 commands.

Reply:
Ah, okay.